src/main/kotlin/com/project/twiliospring/config/TwilioConfiguration.kt

```diff
@@ -16,4 +19,11 @@ class BeanConfiguration(
     fun initTwilio() {
         Twilio.init(twilioSID, twilioToken)
     }
+
+    @Profile("twilio")
+    @Bean(name = ["twilioVerifyService"])
```

Comment:
why do we need bean name? YOu can just rename your method

Reply:
I forgot about this possibility